src/main/yamcs/mdb/oresat0-xtce.xml

```diff
@@ -628,11 +628,13 @@
             <xtce:MetaCommand name="C3FwBank" shortDescription="swap firmware bank">
                 <xtce:ArgumentList>
                     <xtce:Argument name="Bank" argumentTypeRef="Uint32Type"/>
+                    <xtce:Argument name="HMAC32" argumentTypeRef="Uint32Type"/>
                 </xtce:ArgumentList>
                 <xtce:CommandContainer name="C3FwBank">
                     <xtce:EntryList>
                         <xtce:FixedValueEntry binaryValue="02" sizeInBits="8"/>
                         <xtce:ArgumentRefEntry argumentRef="Bank"/>
+                        <xtce:ArgumentRefEntry argumentRef="HMAC32"/>
```

Comment:
What is the HMAC arg for? Isn't this added in the post processor?

Reply:
Hmn, I thought I had removed that before. That predates our discussion on the post-processor.